specification/deviceregistry/resource-manager/readme.md

```diff
@@ -69,10 +69,4 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-go
-  - repo: azure-sdk-for-java
 ```
```

Comment:
@v-hongli1 You left the yaml incorrect. Please revert all changes to Java, we don't want to block other languages again..

Reply:
@XiaofeiCao 
Roll back all changes to before `remove java swagger config`?

Reply:
Just revert this commit https://github.com/Azure/azure-rest-api-specs/pull/34242/commits/0858faa965d694793386c33b5ded46b3f657b1df